src/material-examples/chips-input/chips-input-example.ts

```diff
@@ -17,7 +17,7 @@ export class ChipsInputExample {
   addOnBlur: boolean = true;
 
   // Enter, comma
-  separatorKeysCodes = [ENTER, COMMA];
+  readonly separatorKeysCodes = [ENTER, COMMA];
```

Comment:
Is there a strong reason to do this in the example?

Reply:
In the example, it looks like the key codes are essentially constants, so I think it would make sense if we add the readonly modifier. The motivation for this change proposal came from a teammate who copy pasted the code from the example and interpreted the example code as best practices (i.e. omitting the readonly).